builder/solver.go

```diff
@@ -29,6 +30,8 @@ type solver struct {
 	remoteCache string
 }
 
+var printingMutex sync.Mutex
```

Comment:
This should no longer be necessary, right?

Reply:
Yes, I seem t have forgotten to remove that! I'll clean these minor fixes by the next commit.